import-export-cli/integration/apim/apim.go

```diff
@@ -930,6 +932,27 @@ func (instance *Client) GetApplication(appID string) *Application {
 	return &appResponse
 }
 
+// GetApplications : Get Applications list from APIM
+func (instance *Client) GetApplications() *ApplicationList {
```

Comment:
There are multiple places with unnecessary newlines. Shall we remove them?

Reply:
Fixed with c672258dbfed4a533e8a19dc8f1ef5050d4bd02e